### PR TITLE
Atomic graph mutation

### PR DESCRIPTION
**Pull request by Thomas-git**
Here is a small patch to fix schema with graph use (as seen in Objection [doc](https://vincit.github.io/objection.js/api/query-builder/other-methods.html#context))

Biggest patch is about atomic transaction with upsert/insert graph and with multi create.
I added a $startTransaction param. But Maybe we should name it $atomic ?

### CODE REVIEW

**Comment by dekelev** [comment]
Thank you! I'll review this later this week.

בתאריך יום ד׳, 21 באוק׳ 2020, 12:59, מאת Thomas-git ‏<
notifications@github.com>:

**Comment by dekelev** [comment]
@Thomas-git The PR looks very promising! please add the new operator to the README file.

File: src/index.js [hunk 1]
```diff
@@ -314,12 +314,47 @@ class Service extends AdapterService {
...
+  }
+
   _createQuery (params = {}) {
     const trx = params.transaction ? params.transaction.trx : null;
     const schema = params.schema || this.schema;
     const query = this.Model.query(trx);
-
-    return schema ? query.withSchema(schema) : query;
+    if (schema) {
+      query.context({
```
[the remark this answers is not in the document]
Comment (Thomas-git) [reply src/index.js:351]: I personally don't use schema. I just read that in Objection [doc](https://vincit.github.io/objection.js/api/query-builder/other-methods.html#context)

**Comment by Thomas-git** [comment]
I'll made the changes you suggested later this week. Thanks for reviewing.

**Comment by Thomas-git** [comment]
OK, changes made. I do prefer $atomic, it's easy to understand what it does.

**Comment by dekelev** [comment]
Released with v6.2.0

**Comment by dekelev** [comment]
I'm just changing $atomic to atomic to be consistent with the rest of the params operators naming convention.

**Comment by Thomas-git** [comment]
Ok for `atomic` instead of `$atomic`. Thanks for merging.

**Comment by Thomas-git** [comment]
There is a problem with moving `$atomic` from params.query to `atomic` in params: In params, `atomic` is not sent for remote call.
And, at least in my case `atomic` is most useful on remote call because you can't sent a transaction object remotely.
Do you know a way for params to be forwarded in remote service call ? Else, I think we should move `atomic` back to `$atomic` in params.query

What do you think about it ?

**Pull request merged**